aries_cloudagent/askar/profile.py

```diff
@@ -54,6 +54,15 @@ def store(self) -> Store:
         """Accessor for the opened Store instance."""
         return self.opened.store
 
+    def opened(self) -> AskarOpenStore:
```

Comment:
Doesn't this method clash with the `opened` instance property?

Reply:
Yes, It is not needed.  Removed